deploy-manage/manage-spaces.md

```diff
@@ -23,4 +26,106 @@ $$$spaces-control-feature-visibility$$$
+
+::::{note}
+In a serverless project, you must have an admin role on that project to manage its **Spaces**.
+::::
+
+
+## Create a space [spaces-managing]
+
+$$$spaces-control-feature-visibility$$$
+You can have up to 1,000 spaces by default. The maximum number of spaces can be configured by the `xpack.spaces.maxSpaces` setting (refer to [Spaces settings in {{kib}}](https://www.elastic.co/guide/en/kibana/current/spaces-settings-kb.html)).
+
+::::{note}
+In a serverless project, the number of spaces is limited to 100.
+::::
+
+1. Select **Create space** and provide a name, description, and URL identifier.
+
+    The URL identifier is a short text string that becomes part of the {{kib}} URL when you are inside that space. {{kib}} suggests a URL identifier based on the name of your space, but you can customize the identifier to your liking. You cannot change the space identifier once you create the space.
+
+2. **\[Not available in serverless]** Select a **Solution view**. This setting controls the navigation that all users of the space will get:
+
+    * **Search**: A light navigation menu focused on analytics and Search use cases. Features specific to Observability and Security are hidden.
+    * **Observability**: A light navigation menu focused on analytics and Observability use cases. Features specific to Search and Security are hidden.
+    * **Security**: A light navigation menu focused on analytics and Security use cases. Features specific to Observability and Search are hidden.
+    * **Classic**: All features from all solutions are visible by default using the classic, multilayered navigation menus. You can customize which features are visible individually.
+
+3. **\[Not available in serverless]** If you selected the **Classic*** solution view, you can customize the ***Feature visibility** as you need it to be for that space.
+   ::::{note}
+   Even when disabled in this menu, some Management features can remain visible to some users depending on their privileges. Additionally, controlling feature visibility is not a security feature. To secure access to specific features on a per-user basis, you must configure [{{kib}} Security](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-roles.md).
+   ::::
+
+4. Customize the avatar of the space to your liking.
+5. Save your new space by selecting **Create space**.
+
+You can edit all of the space settings you just defined at any time, except for the URL identifier.
+
+{{kib}} also has an [API](https://www.elastic.co/guide/en/kibana/current/spaces-api.html) if you prefer to create spaces programmatically.
+
+
+## Define access to a space [spaces-control-user-access]
+
+::::{note}
+In serverless projects, customizing access to a space is available for the following project types only: [![Elasticsearch](../images/serverless-es-badge.svg "")](/solutions/search.md) [![Security](../images/serverless-sec-badge.svg "")](/solutions/security/elastic-security-serverless.md)
+::::
```

Comment:
I'd consider pulling this out of a note as well

```
Serverless project types: [![Elasticsearch](../images/serverless-es-badge.svg "")](/solutions/search.md) [![Security](../images/serverless-sec-badge.svg "")](/solutions/security/elastic-security-serverless.md)
```

Reply:
yep I think we all agree that using notes for version/deployment type differences isn't a great way to show it